prov/sockets/src/sock_dom.c

```diff
@@ -519,8 +519,8 @@ int sock_domain(struct fid_fabric *fabric, struct fi_info *info,
 	sock_domain->dom_fid.ops = &sock_dom_ops;
 	sock_domain->dom_fid.mr = &sock_dom_mr_ops;
 
-	if (!info || !info->domain_attr ||
-	    info->domain_attr->data_progress == FI_PROGRESS_UNSPEC)
+	if (!info->domain_attr ||
+		info->domain_attr->data_progress == FI_PROGRESS_UNSPEC)
 		sock_domain->progress_mode = FI_PROGRESS_AUTO;
```

Comment:
The normal practice is to align under the opening paren, plus one space.

if (blah blah &&
    more blah blah)

while (blah blah &&
       more blah blah)

function(blah blah &&
         more blah blah)

(not sure if this will render correctly)  But we're not overly strict, so an off-by one isn't a big deal.  That is, using a tab instead of 7 spaces is okay.  This usually occurs when trying to align with "printf(".

printf("helpful text plus some formatting stuff",
        variables_not_quite_aligned)